### PR TITLE
Update the requirements for senior position applicants

### PR DESCRIPTION
**Pull request by jcfausto**
In the past month, we evaluated two code challenges for the Bookings team.

While discussing the evaluations we came to the conclusion that for senior position applicants it would make more sense if we add the "dockerization" of the solution as a requirement. 

Also, we had candidates submitting Ruby solutions with old versions. For this case, we suggested adding the minimum version expected for Ruby (2.6.x) to align with the same version we use internally.

This PR adds those changes to the Backend and FullStach code challenges.

### CODE REVIEW

**Review by adamlukemooney — APPROVED** [review]

**Review by nicolasiensen — APPROVED** [review]

File: fullstack/README.md [hunk 1]
```diff
@@ -75,12 +75,16 @@ All valid position data, that are inside the "city boundaries", must be stored s
...
 * You are free to make use of any frontend framework or library you please (if any), but you should justify your choice.
 
+## Containerized solution
+
+A containerized solution is a requirement for senior position applicants.
+
 ## Optional extras
 
-* A containerized solution is not a requirement, but definitely preferred for us to easily deploy your solution and test. Therefore, adding a Dockerfile to your solution is optional, but recommended.
+* A containerized solution is not a requirement for non senior position applicants, but definitely preferred for us to easily deploy your solution and test. Therefore, adding a Dockerfile to your solution is optional, but recommended.
```
Comment (yonelacort) [inline fullstack/README.md:87]: I find this sentence a bit confusing

_Therefore, adding a Dockerfile to your solution is optional, but recommended._

I'd use instead

_Therefore, adding a Dockerfile is not mandatory, but recommended._

Comment (yonelacort) [inline fullstack/README.md]:
```suggestion
* A containerized solution is not a requirement for non senior position applicants, but definitely preferred for us to easily deploy your solution and test. Therefore, adding a Dockerfile is not mandatory, but recommended.
```

File: backend/README.md [hunk 2]
```diff
@@ -73,12 +73,16 @@ All valid position data, that are inside the "city boundaries", must be stored s
...
 
-## _Optional_ extras
+## Containerized solution
 
-* A containerized solution is not a requirement, but definitely preferred for us to easily deploy your solution and test. Therefore, adding a Dockerfile to your solution is optional, but recommended.
+A containerized solution is a requirement for senior position applicants.
+
+## Optional extras
+
+* A containerized solution is not a requirement for non senior position applicants, but definitely preferred for us to easily deploy your solution and test. Therefore, adding a Dockerfile to your solution is optional, but recommended.
```
Comment (yonelacort) [inline backend/README.md]:
```suggestion
* A containerized solution is not a requirement for non senior position applicants, but definitely preferred for us to easily deploy your solution and test. Therefore, adding a Dockerfile is not mandatory, but recommended.
```

**Comment by jcfausto** [comment]
Thanks for suggestions @yonelacort, they were applied.

**Review by yonelacort — APPROVED** [review]

**Review by tbueno — APPROVED** [review]

**Pull request merged**